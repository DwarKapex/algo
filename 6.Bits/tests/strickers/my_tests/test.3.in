8/8/8/8/8/2B5/8/8
