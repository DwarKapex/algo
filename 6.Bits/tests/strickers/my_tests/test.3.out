0
9241421692918827537
0